plugins/module_utils/fusion.py

```diff
@@ -47,6 +47,7 @@
 
 PARAM_ISSUER_ID = "issuer_id"
 PARAM_PRIVATE_KEY_FILE = "private_key_file"
+PARAM_PRIVATE_KEY_PASSWORD = "private_key_password"
```

Comment:
What about environmental variable. Does ansible collection need it?

Reply:
no, it doesn't as it was decided that python sdk will handle them